downloads.html

```diff
@@ -5,12 +5,53 @@
 	<div class="right">
     <h1>Download</h1>
 
-    <p>2.0.1 is the latest release. The current stable version is 2.0.1.</p>
+    <p>2.1.0 is the latest release. The current stable version is 2.1.0.</p>
 
     <p>
     You can verify your download by following these <a href="https://www.apache.org/info/verification.html">procedures</a> and using these <a href="https://kafka.apache.org/KEYS">KEYS</a>.
     </p>
 
+    <span id="2.1.0"></span>
+    <h3 class="download-version">2.1.0<a href="#2.1.0"><i class="fas fa-link " style="color:#053ce2"></i></a></h3>
+        <ul>
+            <li>
+                Released Nov 20, 2018
+            </li>
+            <li>
+                <a href="https://www.apache.org/dist/kafka/2.1.0/RELEASE_NOTES.html">Release Notes</a>
+            </li>
+            <li>
+                Source download: <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/2.1.0/kafka-2.1.0-src.tgz">kafka-2.1.0-src.tgz</a> (<a href="https://www.apache.org/dist/kafka/2.1.0/kafka-2.1.0-src.tgz.asc">asc</a>, <a href="https://www.apache.org/dist/kafka/2.1.0/kafka-2.1.0-src.tgz.sha512">sha512</a>)
+            </li>
+            <li>
+                Binary downloads:
+                <ul>
+                    <li>Scala 2.11 &nbsp;- <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/2.1.0/kafka_2.11-2.1.0.tgz">kafka_2.11-2.1.0.tgz</a> (<a href="https://www.apache.org/dist/kafka/2.1.0/kafka_2.11-2.1.0.tgz.asc">asc</a>, <a href="https://www.apache.org/dist/kafka/2.1.0/kafka_2.11-2.1.0.tgz.sha512">sha512</a>)</li>
+                    <li>Scala 2.12 &nbsp;- <a href="https://www.apache.org/dyn/closer.cgi?path=/kafka/2.1.0/kafka_2.12-2.1.0.tgz">kafka_2.12-2.1.0.tgz</a> (<a href="https://www.apache.org/dist/kafka/2.1.0/kafka_2.12-2.1.0.tgz.asc">asc</a>, <a href="https://www.apache.org/dist/kafka/2.1.0/kafka_2.12-2.1.0.tgz.sha512">sha512</a>)</li>
+                </ul>
+                We build for multiple versions of Scala. This only matters if you are using Scala and you want a version built for the same Scala version you use. Otherwise any version should work (2.11 is recommended).
```

Comment:
I think we should recommend 2.12 at this point. 2.11 has not been updated in a long time and everyone is moving to 2.12 (even Spark).